transform-plugins/docs/XMLMultiParser-transform.md

```diff
@@ -27,6 +27,11 @@ children of the elements referenced by the XPath. For example: /rss/channel/item
 **schema:** The schema of records to output. Each field in the schema must be a child of the XML element referenced by
 the XPath. Currently only simply types are supported.
 
+Conditions
+----------
+If error dataset is configured then all the erroneous rows, if present in the input, will be committed to the
+specified error dataset. If not configured, the erroneous rows will be committed to default error dataset.
+
```

Comment:
If nothing is specified, it actually doesn't write to any dataset and logs a warning in the logs. There is no "default" error dataset.